README.md

```diff
@@ -45,6 +45,7 @@ custom:
     stages:
       - '[name of the stage to apply log forwarding]'
       - '[another stage name to filter]'
+    disabledLambdaPermission: false # whether to disable creation of the AWS::Lambda::Permission for the destinationARN (when policy size limits are a concern)
```

Comment:
Maybe it's just me, but I wonder if `createLambdaPermission: true` is clearer than the double negative? Leave it up to you...

Reply:
I personally like more `createLambdaPermission: true`